### PR TITLE
fixed IE8 console issues, added minified version

### PR DESCRIPTION
**Pull request by camsjams**
I fixed IE8 console issue, improved console logging, added minified version

### CODE REVIEW

**Comment by rafde** [comment]
I was making this function output messages that were distinct to this class, hence prepending psp. I will make your changes but will want to keep the pre string. Will do when I get back.

File: PrioritizedSubPub.js [hunk 1]
```diff
@@ -206,12 +206,17 @@
      * @desc private console output. Currently logs by default.
      * @todo: think of good way to toggle logging.
      */
-     function _debugLog() {
+    function _debugLog() {
         var args;
-        if (console && console.log) {
+        if (window.console && console.log) {
```
Comment (rafde) [inline PrioritizedSubPub.js:211]: why do you want `console` to only work for browsers? I wrote this with nodejs in mind as well.

Comment (camsjams) [inline PrioritizedSubPub.js:211]: Good point - well it needs to fail gracefully in IE8 as `if(console)` will throw errors.

So perhaps something more elegant that would fit here is something along the lines of
`hasConsole =  window ? (window.console && window.console.log) : (console && console.log)`

but I leave it for you to decide :)

Comment (rafde) [inline PrioritizedSubPub.js:211]: I am going to have to try something like `var console = window.console || console;` and use the var instead. I am going to reopen the issue version and I'll address and close this PR. Thanks for pointing this out.

**Pull request closed without merging**